### ISSUE TITLE
How do I use keepalive?

### ISSUE THREAD

**Issue by hexraven** [hmage/cpuminer-opt#3]
I used to use -k option for TCP keepalive and wondering how to use that with this version.
In version news, there is an entry for keepalive functionality https://github.com/hmage/cpuminer-opt/blob/master/NEWS#L164. I just can't figure it out how to specify it.

Any hint?

**Comment by hmage** [comment]
The version history of cpuminers and their forks is a big hairy ball of mess. If something doesn't work it's better practice, for now, to take it at face value as if it never was there.

As for keepalive, the way I read the source code — it's used by default on stratum connections if libcurl version is higher than 7.16.06. 

https://github.com/hmage/cpuminer-opt/blob/v3.5.0-hmage/util.c#L1172

JSON RPC connections use keepalive if the algorithm in question asks for it by putting a JSON_RPC_LONGPOLL flag and libcurl version is higher than 7.16.06.

https://github.com/hmage/cpuminer-opt/blob/v3.5.0-hmage/util.c#L497

Basically it depends on algorithm for RPC, and on by default on stratum connections.

Hope this answers your question.
